podpac/core/algorithm/stats.py

```diff
@@ -45,6 +45,10 @@ class Reduce(Algorithm):
     _reduced_coordinates = tl.Instance(Coordinates, allow_none=True)
     _dims = tl.List(trait_type=str)
 
+    @tl.default("outputs")
```

Comment:
@mpu-creare This simple update that sets the `outputs` using the `source.outputs` allows the reduce nodes (like Min and Median) to handle nodes with multiple outputs. Should I spend some more time on the other Algorithm nodes updating them as well?

 * I think the Convolution will be the same, so it's probably worth it
 * I think the coordinate selection nodes might be just as easy, as well
 * The Arithmetic node could be a bit more work

I guess the same thing applies to DataSource types and the Compositors. Should I be proactive updating those or do you want to do them as needed?
 * I think I should do OrderedCompositor (or do you want to require that OrderedCompositor sources are always single-output?)
 * I think the ReprojectedSource node will be easy, like the Reduce nodes here
 * We already discussed doing the DatasetCoordinatesMixin, which will cover
    - H5PY
    - Zarr
 * That leaves
    - PyDap
    - CSV
    - Rasterio
    - WCS